### PR TITLE
added proceeds to checkout

### PR DESCRIPTION
**Pull request by gitJanaeW**

### CODE REVIEW

File: client/src/components/shopping/CartItem/index.js [hunk 1]
```diff
@@ -63,7 +68,7 @@ const CartItem = ({ item }) => {
           </span>
         </div>
       </div>
-    </div>
+    </div>) : null
```
Comment (SonjaIlic94) [inline client/src/components/shopping/CartItem/index.js]: remove null

File: client/src/utils/shopping/queries.js [hunk 2]
```diff
@@ -11,7 +11,6 @@ export const QUERY_PRODUCTS = gql`
       image
       category {
         _id
-        name
```
Comment (SonjaIlic94) [inline client/src/utils/shopping/queries.js]: put name back pls

**Review by SonjaIlic94 — CHANGES_REQUESTED** [review]
please remove null & add back "name" to category query.

**Comment by gitJanaeW** [comment]
I'll make another request with the added changes

File: client/src/components/shopping/CartItem/index.js (hunk 1 above)
Comment (SonjaIlic94) [reply client/src/components/shopping/CartItem/index.js]: thx

**Review by SonjaIlic94 — COMMENTED** [review]
looks good!

**Pull request merged**